core/stencil.config.ts

```diff
@@ -9,6 +9,7 @@ import { apiSpecGenerator } from './scripts/api-spec-generator';
 
 export const config: Config = {
   autoprefixCss: true,
+  sourceMap: false,
```

Comment:
There is a bug in stencil that causes sourcemap generation to fail in Ionic. The stencil team is aware of this and they are going to fix it.